podcasts/PlaybackManager.swift

```diff
@@ -2084,13 +2086,16 @@ private extension PlaybackManager {
     }
 
     // MARK: - Transcripts
-    private func loadTranscripts() {
-        guard FeatureFlag.transcripts.enabled, let episode = currentEpisode(), let podcast = currentPodcast else {
+    private func loadTranscripts(episode: BaseEpisode) {
+        guard FeatureFlag.transcripts.enabled, let episode = episode as? Episode, let podcast = episode.parentPodcast() else {
             return
         }
-
+        transcriptsAvailable = false
         Task.init {
-            let _ = try? await ShowInfoCoordinator.shared.loadTranscripts(podcastUuid: podcast.uuid, episodeUuid: episode.uuid)
+            if let transcripts = try? await ShowInfoCoordinator.shared.loadTranscripts(podcastUuid: podcast.uuid, episodeUuid: episode.uuid) {
+                transcriptsAvailable = !transcripts.isEmpty
+                NotificationCenter.postOnMainThread(notification: Constants.Notifications.episodeTranscriptChanged)
```

Comment:
Does this mean the episode transcript **changed** (like available, unavailable, or any other change) or is it available?

Reply:
It's mean that a Transcript is available, so are you suggesting to rename to `episodeTranscriptAvailable` ?

Reply:
Yep!